kotlin/view-point-cloud-data-offline/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,6 @@
     package="com.esri.arcgisruntime.samples.viewpointclouddataoffline">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
just double checking these changes were intended here since this class isn't related to image overlay?